core/strings/ascii_set.odin

```diff
@@ -3,9 +3,22 @@ package strings
 
 import "core:unicode/utf8"
 
+/*
+Ascii_Set is designed to store ASCII characters efficiently as a bit-array
+Each bit in the array corresponds to a specific ASCII character, where the value of the bit (0 or 1) 
+indicates if the character is present in the set or not.
+*/
 Ascii_Set :: distinct [8]u32
+/*
+Creates an Ascii_Set with unique characters from the input string.
 
-// create an ascii set of all unique characters in the string
+Inputs:
+- chars: A string containing characters to include in the Ascii_Set.
+
+Returns:
+- as: An Ascii_Set with unique characters from the input string.
+- ok: false if any character in the input string is not a valid ASCII character.
+*/
```

Comment:
This will be formatted incorrectly on the website.
You need to have 2 trailing spaces in markdown to add a break and not have everything be on a single line.
i.e.
`Inputs:  ` rather than `Inputs:`

Furthermore to make this stand out more on the website it might be beneficial to make the `Inputs`, `Returns`, etc. to be bold.
i.e.
`**Inputs**  `

Note you don't need to do this for `Example:` since that has its own formatting rules when generating the website documentation.